src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/StomperBoots.java

```diff
@@ -89,4 +91,20 @@ private Vector getShockwave(Location origin, Location target) {
         }
     }
 
+    /**
+     * Checks if the stomper boots can move an entity and is not the player who is using the boots.
+     * <br><b>For developers:</b> If you're spawning an immovable NPC, you should be denying
+     * collision with {@code LivingEntity#setCollidable(false)} or
+     * gravity with {@code LivingEntity#setGravity(false)}.
```

Comment:
Why are you using `@code` instead of `@link` here?

Reply:
Just for the sake of adding `false` to be clear what people should do. I can change it